x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/resource_findings/resource_findings_container.tsx

```diff
@@ -107,18 +110,38 @@ export const ResourceFindings = ({ dataView }: FindingsBaseProps) => {
         <EuiSpacer />
         {error && <ErrorCallout error={error} />}
         {!error && (
-          <ResourceFindingsTable
-            loading={resourceFindings.isFetching}
-            items={resourceFindings.data?.page || []}
-            pagination={getPaginationTableParams({
-              pageSize: urlQuery.pageSize,
-              pageIndex: urlQuery.pageIndex,
-              totalItemCount: resourceFindings.data?.total || 0,
-            })}
-            setTableOptions={({ page }) =>
-              setUrlQuery({ pageIndex: page.index, pageSize: page.size })
-            }
-          />
+          <>
+            {resourceFindings.isSuccess && !!resourceFindings.data.page.length && (
+              <FindingsDistributionBar
+                {...{
+                  type: i18n.translate('xpack.csp.findingsResourceTableHeader.tableRowTypeLabel', {
```

Comment:
This ID feels wrong (missing a `.findings` section for instance) please make sure it is algined with @JordanSh work regarding i18n IDs in this page